### DISCUSSION TITLE
Celery Hanging

### DISCUSSION THREAD

**Discussion by jdhoang** [apache/airflow#23811] [Q&A] [answered]
Trying to convert to CeleryExecutor and any attempt to run any Celery component (i.e., Flower, Worker) hangs without any error message.  Running Celery Worker on another host returns Connect to Redis: No such File or Directory.  I've successfully tested connecting to Redis both on webserver where redis is located as well as on remote worker host.  Also tried uninstalling and re-installing Celery without any luck.

Is there any way to turn on debugging to find out where issue may lie?  Thank you for any assistance!

**Comment by pingzh** [comment, marked as answer]
it will be great if you can share more info, like your airflow version, celery version, your `[celery]` config in airflow.cfg.

**Reply by jdhoang** [reply]
Apologies for not mentioning versions:

Airflow: 2.2.3
Celery: 2.1.0

---------------------------------------------------------------
I believe Celery configuration in airflow.cfg were default (other than meta database and Redis connection)
I just noticed I set ssl_key, ssl_cert, ssl_cacert with PKI cert which were used for webserver - is that necessary for Celery?
---------------------------------------------------------------
[celery]
celery_app_name = airflow.executors.celery_executor
worker_concurrency = 16
worker_umask = 0o077
broker_url = redis://127.0.0.1:6379/0
result_backend = db+postgresql://[user]:[password]@127.0.0.1:5432/[database]
flower_host = 0.0.0.0
flower_port = 5555
celery_config_options = airflow.config_templates.default_celery.DEFAULT_CELERY_CONFIG
ssl_key = [.key]
ssl_cert = [.cer]
ssl_cacert = [.cer]

**Reply by jdhoang** [reply]
pingzh: thank you very much for your suggestion on looking at my airflow.cfg file and celery configuration!   Once I removed the ssl settings, it worked!

**Reply by pingzh** [reply]
@jdhoang it will be great if you can mark this as Answered, thanks https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions

**Reply by jdhoang** [reply]
@pingzh markes as Answered.  Thank you again for your assistance!
